store/storetest/team_store.go

```diff
@@ -869,9 +869,9 @@ func testTeamMembersWithPagination(t *testing.T, ss store.Store) {
 	store.Must(ss.Team().SaveMember(m2, -1))
 	store.Must(ss.Team().SaveMember(m3, -1))
 
-	r1 = <-ss.Team().GetTeamsForUserWithPagination(m1.UserId, 0, 1)
-	require.Nil(t, r1.Err)
-	ms := r1.Data.([]*model.TeamMember)
+	members, errTeam := ss.Team().GetTeamsForUserWithPagination(m1.UserId, 0, 1)
+	require.Nil(t, errTeam)
+	ms := members
```

Comment:
there's no need for the 'ms' variable now. just use 'members'

Reply:
or directly set `ms` instead of `members` 2 lines before.